content/html/concepts/semantic-html/terms/progress/progress.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'progress' 
+Description: 'Used to display a progress bar.' 
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags: 
+  - 'Semantic'
+  - 'Tags'
+CatalogContent: 
+  - 'learn-html'
+  - 'path/front-end--engineer-career-path'
+---
+
+The **`<progress>`** tag is a semantic HTML element that is used to a display a progress bar, an indicator that represents the progress a person has made in completing tasks such as a level in a game, question in an online exam, or parts of a [form](https://www.codecademy.com/resources/docs/html/forms).
+
+> **Note:** When using the `max` attribute in the `<progress>` element, the value must be greater than `0` and be a valid floating point number(e.g., 5.3). If the value is not specified, the default value is `1`.
+
+## Syntax
+
+```html
```

Comment:
Can we list the important attributes under syntax? Also show that there's inner text possible?

Reply:
Sure @SSwiniarski